lib/EvalDevToolModulePlugin.js

```diff
@@ -65,11 +65,14 @@ class EvalDevToolModulePlugin {
 						return source;
 					}
 					const content = source.source();
+					const namespace = compilation.getPath(this.namespace, {
+						chunk: chunkGraph.getModuleChunks(module)[0]
```

Comment:
Why `[0]`?

Reply:
There was only one item in the array. Should it be the last index?

Reply:
Not sure, just want t make sure we really fix it :smile: